src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -97,26 +103,72 @@ class ReimbursementAccountPage extends React.Component {
         if (prevProps.network.isOffline && !this.props.network.isOffline) {
             this.fetchData();
         }
+        if (!this.state.hasACHDataBeenLoaded) {
+            // If the ACHData has not been loaded yet, and we are seeing the default data for props.reimbursementAccount
+            // We don't need to do anything yet
+            if (this.props.reimbursementAccount !== ReimbursementAccountProps.reimbursementAccountDefaultProps
+                && !this.props.reimbursementAccount.isLoading) {
+                // If we are here, it is because this is the first time we load the ACHData from the server and
+                // this.props.reimbursementAccount.isLoading just changed to false. From now on, it makes sense to run the code
+                // below updating states and the route, and this will happen in the next react lifecycle.
+                this.hasACHDataBeenLoaded = true;
```

Comment:
```suggestion
```
We don't have such prop, it's only a state variable.

Reply:
Oops, it was a left over of a first implementation. I'll remove it, thanks.